game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -143,10 +144,15 @@ class Cell(val id: Int, player: Player, var position: Vector2 = new Vector2(0f,
     None
   }
 
+  def defineAiState: AIState = {
```

Comment:
Oh wow I'm silly I didn't even realise that this was the attribute we were putting it in....... Forget my comment.